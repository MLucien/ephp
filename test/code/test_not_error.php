<? $a = false; ?>
<?= (!$a) ? 1 ?>
