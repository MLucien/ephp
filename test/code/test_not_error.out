
Parse error: parse error in {{CWD}}/test/code/test_not_error.php on line 2
